skills_ml/algorithms/occupation_classifiers/classifiers.py

```diff
@@ -0,0 +1,112 @@
+import os
+import logging
+import json
+import tempfile
+
+from collections import Counter
+
+from gensim.models import Doc2Vec
+
+from skills_ml.algorithms.string_cleaners import NLPTransforms
+
+from skills_utils.s3 import download
+
+PATHTOMODEL = 'open-skills-private/model_cache/'
+LOOKUP = 'lookup_va_0605.json'
+
+MODEL_NAME = 'gensim_doc2vec_va_0605'
+DOCTAG = 'gensim_doc2vec_va_0605.docvecs.doctag_syn0.npy'
+SYN0 = 'gensim_doc2vec_va_0605.syn0.npy'
+SYN1 = 'gensim_doc2vec_va_0605.syn1.npy'
+
+class SocClassifier(object):
+    """The SocClassifier Object to classify each jobposting description to O*Net SOC code.
+
+    Example:
+
+    from airflow.hooks import S3Hook
+    from skills_ml.algorithms.occupation_classifiers.classifiers import SocClassifier
+
+    s3_conn = S3Hook().get_conn()
+    Soc = SocClassifier(s3_conn=s3_conn)
+    predicted_soc = Soc.classify(jobposting, mode='top')
+    """
+    def __init__(self, model_name=MODEL_NAME, s3_path=PATHTOMODEL, s3_conn=None):
```

Comment:
I would actually rather there not be a default s3 path. That is specific to our infrastructure. It is easier to change infrastructure if we only control these paths from a config file that doesn't get checked into a repository.